task_7/7-1.js

```diff
@@ -0,0 +1,41 @@
+(() => {
+    'use strict';
+    kintone.events.on(['app.record.create.show','app.record.edit.show'], (event) => {
+		event.record.重複禁止項目_文字列.disabled = true;
+		return event;
+	});
```

Comment:
繰り返しになりますが、インテンド整理しましょう！！
```suggestion
    kintone.events.on(['app.record.create.show','app.record.edit.show'], (event) => {
		  event.record.重複禁止項目_文字列.disabled = true;
		  return event;
	  });
```